src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
- [ ] Kindly consider removing the trailing slash because it has no effect on void elements and interacts badly with unquoted attribute values.
```suggestion
    <meta charset="UTF-8">
```